docker-compose-shared.yml

```diff
@@ -0,0 +1,17 @@
+# Docker Compose over ride file for FreeCodeCamp
+#
+# Run with:
+#   docker-compose -f docker-compose.yml -f docker-compose-shared.yml up
+#
+version: "2"
+services:
```

Comment:
Over rides for services, connects them to the shared network created under `networks`.